widdershins/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG NODE_VERSION=lts-buster
+
+FROM circleci/node:${NODE_VERSION}
+
+ARG WIDDERSHINS_VERSION=4.0.1
+
+LABEL version="${WIDDERSHINS_VERSION}"
+LABEL maintainer="shinohara@chatwork.com"
+
+RUN sudo npm install -g widdershins \
+    && sudo apt-get update \
```

Comment:
@tshinowpub 
How about deleting this line?
This is not necessary because we do not `apt-get install`.

Reply:
fix this commit.

https://github.com/chatwork/dockerfiles/pull/788/commits/677894fa50744c4df4fe1ad416d52887b9bed139